narwhal/types/src/primary.rs

```diff
@@ -484,12 +485,7 @@ impl Hash<{ crypto::DIGEST_LENGTH }> for Vote {
     type TypedDigest = VoteDigest;
 
     fn digest(&self) -> VoteDigest {
-        let mut hasher = crypto::DefaultHashFunction::default();
-        hasher.update(Digest::from(self.digest));
-        hasher.update(self.round.to_le_bytes());
-        hasher.update(self.epoch.to_le_bytes());
-        hasher.update(&self.origin);
-        VoteDigest(hasher.finalize().into())
+        VoteDigest(self.digest.0)
```

Comment:
Since we are doing this refactoring, what do you think of renaming the `self.digest` to something that can more accurately capture the meaning, ex `header_digest` , `header_id` etc. Calculating the `digest()` by using a `self.digest` is a tad confusing.

Reply:
Agree, will try to update this.